pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -652,11 +652,11 @@ type ServiceInstanceCredentialStatus struct {
 	// InProgressProperties is the properties state of the
 	// ServiceInstanceCredential when a Bind is in progress. If the current
 	// operation is an Unbind, this will be nil.
-	InProgressProperties *ServiceInstanceCredentialPropertiesState
+	InProgressProperties *ServiceInstanceCredentialPropertiesState `json:"inProgressProperties"`
```

Comment:
shouldn't it also have an `omitempty`? otherwise it will be initialized with empty string if the value is missing, I believe.

Reply:
Sorry, checked, all good: https://play.golang.org/p/urfzWq1XSY

Reply:
They should have been omitEmpty, sorry.